app/models/support/requests/change_existing_user_request.rb

```diff
@@ -0,0 +1,49 @@
+module Support
+  module Requests
+    class ChangeExistingUserRequest < Request
+      attr_accessor :requested_user,
+                    :additional_comments
+
+      validate do |request|
+        if request.requested_user && !request.requested_user.valid?
+          errors.add :base, message: "The details of the user in question are either incomplete or invalid."
+        end
+      end
+      validates :requested_user, presence: true
```

Comment:
Should the "What do you want to change?" field also be required for this form?
```suggestion
      validates :requested_user, :additional_comments, presence: true
```

Reply:
There's a [validation in the view(https://github.com/alphagov/support/blob/56b720f589447518211a237ddf4d6cf97153538a/app/views/change_existing_user_requests/_requested_user_details.html.erb#L20) 
```
<%= r.email_field :email, required: true, aria: { required: true }, class: "form-control" %>
```
I think the model validation might be coming from `app/models/support/gds/requested_user.rb`

Reply:
I've been asked to make the `additional_comments` textarea fields on both the new forms required - so I'll do that now.

Reply:
Addressed in https://github.com/alphagov/support/pull/1244/commits/87080ed5e81d3ea952a90a30d67c99dc5d37078d